src/collider/src/collider.cpp

```diff
@@ -268,33 +271,24 @@ Collider::Collider(const VecDH<Box>& leafBB,
  */
 template <typename T>
 SparseIndices Collider::Collisions(const VecDH<T>& querriesIn) const {
-  int maxOverlaps = querriesIn.size() * 4;
-  SparseIndices querryTri(maxOverlaps);
-  int nOverlaps = 0;
-  while (1) {
```

Comment:
It's certainly nice to remove this; I use similar logic in `LevelSet`; I'm curious if the same applies, or if it becomes a perf problem when the SDF function is computationally heavy.